website/app/domain/contracts/luma.ts

```diff
@@ -0,0 +1,64 @@
+export type LumaEvent = {
```

Comment:
like Pocketbase, Luma is infra I guess, but somehow I put the type in Contracts because one day, it should be an abstraction of Luma. (EventAStore or something)

I don't disagree, pocketbase, and luma in the domain feel weird too.

But well, before there was any domain at all :)